src/rest-server/src/config/paiConfig.js

```diff
@@ -32,12 +32,14 @@ try {
 
 let paiConfigData = {
     machineList: paiMachineList,
+    debuggingReservationSeconds: Number(process.env.DEBUGGING_RESERVATION_SECONDS),
```

Comment:
To make the rest-server could be run separately, I think it's better to give a default in this stage.
```suggestion
    debuggingReservationSeconds: Number(process.env.DEBUGGING_RESERVATION_SECONDS || 604800),
```

Reply:
Done